app/popup.js

```diff
@@ -477,6 +484,23 @@ class OptionsTab extends React.Component {
           </tbody>
         </table>
 
+        <h4 className="page-header">Import / Export</h4>
+        <div className="row">
+          <div className="col-xs-8">
+            <Button label='Export' clickHandler={_exportData} className='glyphicon-export'/>
+            <Button label='Import' clickHandler={() => {this.fileselector.click()}} className='glyphicon-import'/>
+            <input id="fileselector" type="file" onChange={_importData} ref={(input) => {this.fileselector = input}}/>
+          </div>
+          <div className="col-xs-8">
+            <p className="help-block">
+              Export all information about wrangled tabs. This is a convenient way to restore an old state after reinstalling the extension.
```

Comment:
I'll enable a lint rule for this, but let's keep lines to 100 characters. I can fix it after this PR is merged.